### PR TITLE
- work on LVM thin provisioning

### PR DESCRIPTION
**Pull request by aschnell**
For https://trello.com/c/htNoDhHR/369-5-libstorage-ng-lvm-thin-provisioning-probing.

### CODE REVIEW

File: integration-tests/lvm/delete-thin.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/python
+
+# requirements: lvm volume group test with thin-pool thin-pool and thin thin1
```
Comment (joseivanlopez) [inline integration-tests/lvm/delete-thin.py]: The comment seems to have replacement errors.

File: storage/Devices/DmRaidImpl.cc [hunk 2]
```diff
@@ -161,7 +161,7 @@ namespace storage
 
 	const File rotational_file = prober.get_system_info().getFile(SYSFSDIR + get_sysfs_path() +
 								      "/queue/rotational");
-	rotational = rotational_file.get_int() != 0;
+	rotational = rotational_file.get<int>() != 0;
```
Comment (joseivanlopez) [inline storage/Devices/DmRaidImpl.cc]: Why not `rotational = rotational_file.get<bool>();` ?

Comment (aschnell) [reply storage/Devices/DmRaidImpl.cc]: Indeed.

File: integration-tests/lvm/delete-thin.py (hunk 1 above)
Comment (aschnell) [reply integration-tests/lvm/delete-thin.py]: It the type and the name.

Comment (joseivanlopez) [reply integration-tests/lvm/delete-thin.py]: Ahh ok, I did not realize. Thanks

**Pull request merged**